dev-php/pecl-parallel/pecl-parallel-1.1.4.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PHP_EXT_NAME="parallel"
+PHP_EXT_INI="yes"
+PHP_EXT_ZENDEXT="no"
+
+USE_PHP="php7-3 php7-4 php8-0"
+
+inherit php-ext-pecl-r3
+
+SRC_URI="${SRC_URI} -> ${P}.tgz"
```

Comment:
Not exactly what I had in mind, but hey, if it works it's fine.

(I'd just replicated the eclass line, 
`SRC_URI="https://pecl.php.net/get/${PHP_EXT_NAME}-${PV/_/}.tgz"`

Reply:
Your call.  Or perhaps at least a comment above:

```
# php-ext-pecl will set SRC_URI to the download URI, so just rename
```